assets/src/scss/base/_mixins.scss

```diff
@@ -233,3 +233,26 @@ $nested-comment-left-margin: 50px;
     }
   }
 }
+
+@mixin scrollbar-layout {
+  // Only applied to Firefox
+  scrollbar-width: thin;
+  scrollbar-color: transparent;
+
+  &::-webkit-scrollbar {
+    width: $sp-1;
```

Comment:
I don't think we should use these variables for `width` (nor for `border-radius` below)

Reply:
You're right. Sorry I copied this from the other navbar's PR